components/frontend/src/source/SourceParameters.test.js

```diff
@@ -16,19 +16,36 @@ function renderSourceParameters({placeholder = "", type = "string", source_param
                                 type: type,
                                 name: "Parameter",
                                 metrics: ["violations"]
+                            },
+                            other_parameter_key: {
+                                default_value: "Other parameter default",
+                                placeholder: "Other parameter placeholder",
+                                type: type,
+                                name: "Other parameter",
+                                metrics: ["violations"]
+                            },
+                        },
+                        parameter_layout: {
+                            "location": {
+                                "name": "Location parameters",
+                                "parameters": ["parameter_key"]
+                            },
+                            "other": {
+                                "name": "Other parameters",
+                                "parameters": []
                             }
                         }
                     }
                 }
             }
         }>
             <SourceParameters
-                report={{subjects: {}}}
-                metric={{type: "violations"}}
-                source={{ type: "source_type", parameters: {parameter_key: source_parameter_value} }}
+                report={{ subjects: {} }}
+                metric={{ type: "violations" }}
+                source={{ type: "source_type", parameters: { parameter_key: source_parameter_value } }}
                 changed_param_keys={changed_param_keys}
             />
-        </DataModel.Provider>
+        </DataModel.Provider >
```

Comment:
Why the additional space?

Reply:
Hmm, inserted by accident apparently and it stayed there because formatter in VS Code does not remove it.